.infrastructure/env/prod/terraform.tfvars

```diff
@@ -1,3 +1,4 @@
 aws_region        = "eu-south-1"
 environment       = "prod"
 github_repository = "pagopa/b2b-portals"
+dns_domain_name   = "b2b.pagopa.it" ## Change this value
```

Comment:
```suggestion
```

Reply:
with next commit I comment the lines now to be deleted but necessary as soon as we have the domain and certificate

Reply:
changes present in the latest commit